.npmignore

```diff
@@ -20,9 +20,25 @@ lib-cov
 # Coverage directory used by tools like istanbul
 coverage
 
-# Builds
+# Builds and tooling
 .grunt
+Gruntfile.js
 .sass-cache
+.sass-lint.yml
+.travis.yml
+.babelrc
+.codeclimate.yml
+.dockerignore
+Dockerfile
+.all-contributorsrc
+.all-contributrsrc.json
+.eslintcache
+.eslint-babel.json
+.eslintrc.js
```

Comment:
have you considered moving some of the .files into package.json? 

I've found that provides a better picture when trying to understand the project.

Reply:
Makes sense, will add a ticket